ens-de-lyon-centre-d-ingenierie-documentaire.csl

```diff
@@ -173,9 +173,9 @@
             <date-part name="year"/>
           </date>
           <group>
-            <text term="volume" form="short" suffix="."/>
-            <text variable="volume" prefix=". " suffix="/"/>
-            <text variable="number-of-volumes"/>
+              <text term="volume" form="short" suffix=".&#160;"/>
+            <text variable="volume" />
```

Comment:
please fix the spacing here (you added an extra space at the beginning of the line.